lib/MetaCPAN/Server.pm

```diff
@@ -3,6 +3,9 @@ package MetaCPAN::Server;
 use Moose;
 
 ## no critic (Modules::RequireEndWithOne)
+use Catalyst qw/
+    +MetaCPAN::Role::Fastly::Catalyst
+    /;
```

Comment:
No need to fix now, but it would be nice if we were consistent with quoting here.

```perl
use Catalyst qw( +MetaCPAN::Role::Fastly::Catalyst );
```